src/main/java/com/sahil/SplitwiseApp/controller/UsersController.java

```diff
@@ -0,0 +1,26 @@
+package com.sahil.SplitwiseApp.controller;
+
+import com.sahil.SplitwiseApp.model.Users;
+import com.sahil.SplitwiseApp.service.UsersService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("SplitwiseApp")
```

Comment:
If some values are used repeatedly we usually define constant for it, centrally, and use that constant at all places